CONTRIBUTING.md

```diff
@@ -249,6 +249,8 @@ Must be one of the following:
 The scope could be anything specifying place of the commit change. For example `$location`,
 `$browser`, `$compile`, `$rootScope`, `ngHref`, `ngClick`, `ngView`, etc...
 
+You can use `*` if there isn't a more fitting scope.
```

Comment:
According to my (unofficial) understanding, I would say `*` is for when the change affects more than a single scope.